source/tryGoal/component/tryGoalController.js

```diff
@@ -156,6 +156,11 @@ function($scope, agentMemory, $compile, $stateParams, config, GoalAPI, ConfigAPI
 							}
 						})
 					});
+					question.concepts.forEach(function (concInst) {
```

Comment:
technically this can never happen but in the future we might push object side concept instances that aren't a valid response.